Pradeo/pradeo-mtd/ingest/parser.yml

```diff
@@ -0,0 +1,184 @@
+name: pradeo-mtd
+pipeline:
+  - name: json_event
+    external:
+      name: json.parse-json
+  - name: metadata
+  - name: initiator
+  - name: permission
+  - name: detection
+  - name: correlation
+  - name: compliance
+stages:
+  metadata:
+    actions:
+      - set:
+          pradeo.metadata.id: '{{json_event.message.id}}'
+          pradeo.metadata.creationDate: '{{json_event.message.creationDate}}'
+          pradeo.metadata.source: '{{json_event.message.source}}'
+          pradeo.metadata.category: '{{json_event.message.category}}'
```

Comment:
From `json_event.message.category`, can you set ECS fields `event.category` and `event.type` (both as arrays) according to the [ECS categorization](https://www.elastic.co/guide/en/ecs/8.11/ecs-allowed-values-event-category.html)? Thanks